spec/extensions/fourier_transform_functions.md

```diff
@@ -0,0 +1,506 @@
+# Fourier transform Functions
+
+> Array API specification for Fourier transform functions.
+
+A conforming implementation of the array API standard must provide and support the following functions adhering to the following conventions.
+
+-   Positional parameters must be [positional-only](https://www.python.org/dev/peps/pep-0570/) parameters. Positional-only parameters have no externally-usable name. When a function accepting positional-only parameters is called, positional arguments are mapped to these parameters based solely on their order.
+-   Optional parameters must be [keyword-only](https://www.python.org/dev/peps/pep-3102/) arguments.
+-   Broadcasting semantics must follow the semantics defined in {ref}`broadcasting`.
+-   Unless stated otherwise, functions must support the data types defined in {ref}`data-types`.
+-   Unless stated otherwise, functions must adhere to the type promotion rules defined in {ref}`type-promotion`.
+-   Unless stated otherwise, floating-point operations must adhere to IEEE 754-2019.
+
+## Objects in API
+
+<!-- NOTE: please keep the functions in alphabetical order -->
```

Comment:
Please don't sort alphabetically...For other modules it may be fine, but for FFT I expect to see `ifft` follows immediately after `fft` and so on, so that we can make a direct comparison between forward and backward transforms. During the review I had to jump back and forth to check things 😢

Reply:
I'll change this! It was also a little bit weird for me writing the extension alphabetically but I also agree that it is really hard to read